modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItBigZoneOperationTest.java

```diff
@@ -30,7 +30,6 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 
 @WithSystemProperty(key = IgniteSystemProperties.COLOCATION_FEATURE_FLAG, value = "true")
-// TODO: https://issues.apache.org/jira/browse/IGNITE-24943 - remove the mute.
 @ExtendWith(FailureManagerExtension.class)
 @MuteFailureManagerLogging
```

Comment:
Is the mute still needed?

Reply:
I don't know, why was it there in the first place?

Reply:
It was there to avoid those 'Critical system failure' logs due to FailureProcessor being triggered because of the bug you are fixing here :) If the bug is fixed, the mute is not needed anymore (lines 33 and 34 can be removed). The TODO was specifically about this.

Reply:
you are right, I've read the TODO, but not the text =(